apps/DisplayCluster/TouchMouseArea.cpp

```diff
@@ -69,7 +69,7 @@ bool TouchMouseArea::sceneEvent( QEvent* event_ )
 void TouchMouseArea::mousePressEvent( QGraphicsSceneMouseEvent* event_ )
 {
     _mousePressPos = event_->scenePos();
-    emit touchBegin();
```

Comment:
Without the touchBegin() event, does the window move to the front when clicked? Also, I wondering about this pan event... any idea why the one in the mouseMoveEvent is not enough? Because this will generate a pan event when clicking in-place, and I am not sure this is correct...